app/routes/licence.routes.js

```diff
@@ -4,6 +4,18 @@ const LicencesController = require('../controllers/licences.controller.js')
 
 const routes = [
   {
+    method: 'GET',
+    path: '/licences/{id}',
+    handler: LicencesController.summary,
+    options: {
+      auth: {
+        access: {
+          scope: ['billing']
+        }
+      },
+      description: 'Licence summary tab'
+    }
```

Comment:
Based on the comment about changing the handler it would need to be changed here as well.

```suggestion
    method: 'GET',
    path: '/licences/{id}',
    handler: LicencesController.view,
    options: {
      auth: {
        access: {
          scope: ['billing']
        }
      },
      description: 'View a licence page'
    }
```